src/datumaro/plugins/data_formats/tabular.py

```diff
@@ -84,45 +86,56 @@ def _parse(
         items: List[DatasetItem] = []
         categories: TabularCategories = TabularCategories()
 
+        if target is not None:
+            target = eval(",".join(target)) if isinstance(target, list) else target
+            if target["input"] is None or target["output"] is None:
+                raise TypeError('Target should have both "input" and "output"')
```

Comment:
maybe it is more good for considering `target={}`. In this case, media=TableRow(all_data) and annotations=[]. What do you think?

Reply:
Thank you